src/main/java/se/sundsvall/supportmanagement/api/model/errand/TimeMeasurement.java

```diff
@@ -0,0 +1,120 @@
+package se.sundsvall.supportmanagement.api.model.errand;
+
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "Time measure model")
+public class TimeMeasurement {
+
+	@Schema(description = "Status for the time measurement period", example = "Started")
+	private String status;
+
+	@Schema(description = "Start time for the time measurement period", example = "2021-09-01T12:00:00Z")
+	private OffsetDateTime startTime;
+
+	@Schema(description = "Stop time for the time measurement period", example = "2021-09-01T13:00:00Z")
+	private OffsetDateTime stopTime;
+
+	@Schema(description = "Description of the time measurement period", example = "Suspected while waiting for answer from customer")
+	private String description;
+
+	@Schema(description = "Administrator for the time measurement period", example = "JO12DOE")
+	private String administrator;
```

Comment:
Might want to be readOnly, should never be used to patch an errand right?

Reply:
Not exposed in the Errand API-model. Only accessible via a getmapping. But I guess it couldn't hurt.